src/accelerate/components/ListItem.tsx

```diff
@@ -131,7 +135,10 @@ const ListItem = function ( props: {
 						</div>
 					</div>
 					<div className='record-name__title'>
-						<a href={ post.url || post.editUrl || '' } onClick={ () => trackEvent( listId, 'Navigate', { type: post.type } ) }>
+						<a href={ post.url || post.editUrl || '' } onClick={ (e: React.MouseEvent) => {
+							trackEvent( listId, 'Navigate', { type: post.type } );
```

Comment:
This will trigger even if the href is empty and there's no navigation happening. I think it would be best to not have the anchor tag at all if there's no URL.